heka/sandbox/filters/telemetry_s3output_monitors.lua

```diff
@@ -17,11 +17,23 @@ Config:
     ticker_interval = 60
     preserve_data = false # should always be reset on Heka restarts
     message_matcher = "Type == 'heka.all-report'"
+    [TelemetryS3OutputMonitors.config]
+    # CSV to ignore low volume streams
```

Comment:
Oh, you mean ignore_stalls = "A,B,C".  Might be good to include multiple items to make this clearer.